Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Grid-Editor/index.md

```diff
@@ -0,0 +1,670 @@
+
+### Allowance
+
+
+- **Allow in root color** - Determine wether this Block can be created in the root of your layout.
+- **Allow in area** - Determine wether this Block can be created inside Areas of other Blocks.
+
+
+### Areas
+
+Block Areas enables for nesting of Blocks.
+The each area has a size, defined by column and rows spans.
+The grid for the Areas are based on the same amount of columns as your root grid, unless you choose to change it.
+To scale a Area drag the little scale-button in the bottom right corner with your mouse.
+
+- **Grid Columns for Areas** - Overwrite the amount of columns used for the grid of Areas.
+- **Allow in area** - Determine wether this Block can be created inside Areas of other Blocks.
+
+### Area configuration
+
+> Missing screenshot of Area configuration
+
+**MISSING DESCRIPTION OF AREA CONFIG**
+Mention that the alias can be used to target this Area via the Layout Stylesheet.
+
+**Create Button Label** is to overwrite the Create Button Label of that Area.
+
+**Number of blocks** is for the total amount of Blocks in the Area.
+
+**TODO...**
+Mention that you can pick groups for allowance, in that case the required number is for Blocks of the group.
+Mention that you can pick individual Blocks, this Block can be picked and will be valid though its not allow for Areas.
+Individual Block validation(required number) is just for that block-type.
+
+
+### Advanced
+
+These properties are relevant when you working with custom views or complex projects.
+
+- **Custom view** - Overwrite the AngularJS view for the block presentation in the Content editor. Use this to make a more visual presentation of the block or make your own editing experience by adding your own AngularJS controller to the view.
+
+- **Custom stylesheet** - Pick your own stylesheet to be used for this block in the Content editor. By adding a stylesheet the styling of this block will become scoped. This means that the default backoffice styles are no longer present for the view of this block.
+
+- **Overlay editor size** - Set the size for the Content editor overlay for editing this block.
+
+- **Hide content editor** - Use this feature when you want to hide the ability to edit the content in a Block Editor. If you made a custom view that enables you to edit the content part of a Block, you might want to use this feature.
+
+## Editing Blocks
+
+When viewing a **Block Grid** editor in the Content section for the first time, you will be presented with the option to Add content.
+
+![Block List - Add Content](images/BlockListEditor_AddContent.png)
+> TODO: screenshot
+
+Clicking the Add content button brings up the **Block Catalogue**.
+
+![Block List - Setup](images/BlockListEditor_BlockPicker_simplesetup.jpg)
+> TODO: screenshot
+
+The Block Catalogue looks different depending on the amount of available Blocks and their catalogue appearance.
```

Comment:
Don't forget to mention or update the screenshot as blocks can be organised into groups

Reply:
All screenshots will be updated before we publish - hence all the "todo's" 😄